src/psyclone/transformations.py

```diff
@@ -2902,10 +3145,15 @@ def make_constant(symbol_table, arg_position, value,
             else:
                 print("    Modified {0}, arg position {1}, value {2}."
                       "".format(orig_name, arg_position, value))
+        # --------------------------------------------------------------------
```

Comment:
Please remove this line.

Reply:
I am strongly against that - I found it impossible to see where the internal function definition for ``make_constant`` ends - it's nearly 60 lines of code!
If you really don't like this, I would suggest to either add an explicit "# end of make_constant" comment (or replace #--- with the above comment), or  move ``make_constant`` out of the test function to be a separate function.

Reply:
Ah, I hadn't even spotted it was an internal function. I agree with you in that case.